packages/frontend/editor-ui/src/features/dataStore/components/dataGrid/DataStoreTable.vue

```diff
@@ -148,16 +163,19 @@ watch([dataStoreGridBase.currentSortBy, dataStoreGridBase.currentSortOrder], asy
 	await setCurrentPage(1);
 });
 
+watch(currentFilterJSON, async () => {
```

Comment:
What is the difference between this and `onFilterChanged`?

Reply:
onFilterChanged is handling the updates of the currentFilterJSON and this one is to refetch the data if the filters have changed. It could also be restructured so that this is done at the end of onFilterChanged but this will require adding another prop to the `useDataStoreColumnFilters` so I'm not sure it's necessarily a better approach.